front-end/src/app/components/user-profile/user-profile.component.scss

```diff
@@ -53,3 +53,18 @@
     color: #8f8f8f;
   }
 }
+
+// I HATE CSS
+.username {
+  h1 {
+    display: inline-block;
+    padding: 20px;
+    font-size: 36px;
+    position: relative; // Is this good practice?
```

Comment:
I think the best practise would be to use `vertical-align` on the image, instead of re-positioning the header, but I'm not sure if it will work in that case.

I don't really mind about some CSS being not best practise, so just leave it if it works. However, can you remove these comments?